charts/keycloak/values.yaml

```diff
@@ -261,6 +261,15 @@ ingress:
         - keycloak.example.com
       secretName: keycloak-tls
 
+## Network policies configuration
+networkpolicy:
+  # If true, the Network policies are deployed
+  enabled: false
+
+  # Define all other external allowed source
+  # See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#networkpolicypeer-v1-networking-k8s-io
+  # from :{}
```

Comment:
```suggestion
  from: {}
```

Reply:
Empty and not defined `from` doesn't mean the same thing in network policies. 
That why i do not se a default value for this key and I add a `{{- if .Values.networkpolicy.from }}` in the template.

Reply:
Actuall, it should be a list. I was wrong here. So, the default should be `[]`. It is best practice to always specify a default. this should work with your existing code. Just keep the `if`. Actually, a `with` statement would be preferable here.